File: foundations/py/7/isleapyear.py
def isleapyear(year):
    return ((year % 4) == 0 and (year % 100) != 0) or (year % 400) == 0

if __name__ == "__main__":
    year = int(input('Year:'))
    if isleapyear(year):
        print('Year {0} was a leap year'.format(year))

File: 6233-OS/02/isleapyear.c
#include <stdio.h>
#include <stdlib.h>
#include <stdbool.h>

int main(int argc, char *argv[])
{
	int year = atoi(argv[1]);
	bool isLeap = year % 4 == 0 && (year % 100 != 0 || year % 400 == 0);
	printf("%i was %s leap year\n", year, isLeap ? "a" : "not a");
	return 0;
}